src/index.js

```diff
@@ -46,28 +57,55 @@ function setup(webContents) {
         config.set('credentials', credentials);
         // Save senderId
         config.set('senderId', senderId);
-        // Notify the renderer process that the FCM token has changed
-        webContents.send(TOKEN_UPDATED, credentials.fcm.token);
+        // Notify the renderer processes that the FCM token has changed
+        send(TOKEN_UPDATED, credentials.fcm.token);
       }
       // Listen for GCM/FCM notifications
       await listen(Object.assign({}, credentials, { persistentIds }), onNotification(webContents));
-      // Notify the renderer process that we are listening for notifications
-      webContents.send(NOTIFICATION_SERVICE_STARTED, credentials.fcm.token);
+      // Notify the renderer processes that we are listening for notifications
+      send(NOTIFICATION_SERVICE_STARTED, credentials.fcm.token);
+      started = true;
     } catch (e) {
       console.error('PUSH_RECEIVER:::Error while starting the service', e);
-      // Forward error to the renderer process
-      webContents.send(NOTIFICATION_SERVICE_ERROR, e.message);
+      // Forward error to the renderer processes
+      send(NOTIFICATION_SERVICE_ERROR, e.message);
+      starting = false;
```

Comment:
`starting` is never set to false if no error is thrown. Is it intended ?

Reply:
Yes - in that case `started` should be true, so it would return around line 44 anyway.